tests/platform/cloud/AcceptanceTests_oauth2_gpiiAppInstallation.js

```diff
@@ -0,0 +1,233 @@
+/*!
+Copyright 2017 OCAD university
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    jqUnit = fluid.registerNamespace("jqUnit");
+
+fluid.require("%universal");
+
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.cloud.oauth2.gpiiAppInstallation");
+
+var initialTokenResponse;
```

Comment:
Would there be somewhere else that we could track this value, such as in the `{testCaseHolder}`, rather than in a global?